packages/components/src/components/DistributionChart.tsx

```diff
@@ -83,18 +83,26 @@ export const DistributionChart: React.FC<DistributionChartProps> = (props) => {
       }
     }
 
-    const spec = buildVegaSpec(props);
+    const domain = shapes.value.flatMap((shape) =>
+      shape.discrete.concat(shape.continuous)
+    );
+
+    const spec = buildVegaSpec({
+      ...props,
+      minX: props.minX ?? Math.min(...domain.map((x) => x.x)),
+      maxX: props.minX ?? Math.max(...domain.map((x) => x.x)),
+      maxY: Math.max(...domain.map((x) => x.y)),
+    });
```

Comment:
This looks like a new feature or refactoring (I'm unclear to me if this does anything new), was it necessary for testing for some reason?

Reply:
This removes a warning about infinite extents. What I'm doing is explicitly outlining the domain that I want to graph rather than have vega calculate it for me. I don't fully understand why, but this fixes the warning